test/python/scenarios.json

```
@@ -48,6 +48,15 @@
             }
         }
     },
+    "install_additional_python_alpine": {
+        "image": "mcr.microsoft.com/devcontainers/base:alpine",
+        "features": {
+            "python": {
+                "version": "3.11",
+                "additionalVersions": "3.8,3.9.13,3.10.5"
```

```suggestion
                "additionalVersions": "3.10"
```

I like your thinking here, however, it would take a lot of time for the tests runs to finish. Hence, testsing one more version should be fine